### ISSUE TITLE
Game Crash At Level Load

### ISSUE THREAD

**Issue by stkelly85** [Puh00/Resident-Evil-5-Trainer#1]
When running the trainer while loading a level the game will crash to desktop upon completion of the load. The load screen will appear for some time and run normally then the crash occurs. Have not tested to see if all options for trainer cause the crash, but God Mode for sure does. Happens in both Offline mode and Online mode.

Edit: It might be worthwhile to know that this is NOT using the "Gold Edition" of the game.

**Comment by Puh00** [comment]
@stkelly85 
Did you by any chance have problem with the game not launching due to "Games for Windows Live" (GFWL)? This problem is quite common as installing GFWL will often fail. As a matter of fact, when I reinstalled the game just now I wasn't able to launch it due to this problem. So I fixed it by using this [page](http://www.sb-online.org/maluc/index.php?did=bh5fixes), which simply replaces some files in the Resident Evil 5 folder. 

I realized that this GFWL problem made my game crash just as you had described. I think the problem occurs since that solution replaces the "re5dx9.exe" file (the game), which makes it incompatible with my trainer since it modifies that very file. (I'm assuming that my trainer only works for Steams latest version of Resident Evil 5)

I then tried another solution which didn't replace any existing files but instead inserted a xlive.dll file into the Resident Evil 5 folder (which will act as the GFWL). I could then launch the game and I could no longer reproduce the bug that you mentioned.
* Heres a link to a xlive.dll [https://www.dropbox.com/s/9q0ui6miryroo0v/RE5%20Gold%20-%20XLive.7z?dl=0](https://www.dropbox.com/s/9q0ui6miryroo0v/RE5%20Gold%20-%20XLive.7z?dl=0)

Let me know if you had this problem with GFWL.
